vmdb/spec/models/service_template_provision_request_spec.rb

```diff
@@ -99,7 +101,7 @@
 
     it "finished with errors state" do
       @task_1_1.update_and_notify_parent({:state => "finished", :status => "Error", :message => "Error Message"})
-      @task_2_1.update_and_notify_parent({:state => "finished", :status => "Ok", :message => "Test Message"})
+      @task_2_1.update_and_notify_parent(:state => "finished", :status => "Error", :message => "Test Message")
```

Comment:
@tinaafitz 
Does this mean both the children of task 1 have to have :status => "Error" to get the Error Message, previously child 1 had Error and child 2 was Ok